tests/steps/other_v2_steps.py

```diff
@@ -1462,47 +1466,50 @@ def simulate_atc(context):
     'the simulation should report a failure at group "{group}", path "{path}" with message "{message}"'
 )
 def simulate_atc_failure(context, group, path, message):
-    resp: SimulateAtomicTransactionResponse = (
-        context.atomic_transaction_composer_return
-    )
+    if hasattr(context, "simulate_response"):
+        resp = context.simulate_response
+    else:
+        resp = context.atomic_transaction_composer_return.simulate_response
     group_idx: int = int(group)
     fail_path = ",".join(
-        [
-            str(pe)
-            for pe in resp.simulate_response["txn-groups"][group_idx][
-                "failed-at"
-            ]
-        ]
+        [str(pe) for pe in resp["txn-groups"][group_idx]["failed-at"]]
     )
-    assert resp.would_succeed is False
     assert fail_path == path
-    assert message in resp.failure_message
+    assert message in resp["txn-groups"][group_idx]["failure-message"]
 
 
-@when("I prepare the transaction without signatures for simulation")
-def step_impl(context):
-    context.stx = transaction.SignedTransaction(context.txn, None)
+@when("I make a new simulate request.")
+def make_simulate_request(context):
+    context.simulate_request = SimulateRequest(txn_groups=[])
 
 
-@then(
-    'the simulation should report missing signatures at group "{group}", transactions "{path}"'
-)
-def check_missing_signatures(context, group, path):
-    if hasattr(context, "simulate_response"):
-        resp = context.simulate_response
-    else:
-        resp = context.atomic_transaction_composer_return.simulate_response
+@then("I allow more logs on that simulate request.")
+def allow_more_logs_in_request(context):
+    context.simulate_request.allow_more_logs = True
 
-    group_idx: int = int(group)
-    tx_idxs: list[int] = [int(pe) for pe in path.split(",")]
 
-    assert resp["would-succeed"] is False
+@then("I attach the simulate request to simulate the transaction group.")
```

Comment:
nit: maybe "use" instead of "attach" here? Saying this because at first glance, attaching something does not necessarily imply executing the simulation

Reply:
see [renaming a step](https://github.com/algorand/py-algorand-sdk/pull/469/commits/eb40f7226619c571800dacd91009f186cb9c8b4a)